app/forms/idv/finance_form.rb

```diff
@@ -0,0 +1,46 @@
+module Idv
+  class FinanceForm
+    include ActiveModel::Model
+
+    FINANCE_PARAMS = [:ccn, :mortgage, :home_equity_line, :auto_loan].freeze
+
+    attr_reader :idv_params, :finance_type, :finance_account
+
+    validates :finance_type, :finance_account, presence: true
+
+    validate :finance_type_valid
+
```

Comment:
That's a good idea. I'm trying to keep new features (like validation) out of this PR per @sbc100 request. I will add it in a separate PR.